ipstack/plugin.spec.yaml

```diff
@@ -82,7 +84,7 @@ actions:
         required: false
         description: Region name
       time_zone:
-        title: Public IP Address in Decimal
+        title: Public IP Address In Decimal
```

Comment:
Title changes require a major version bump because if a workflow uses this variable and the plugin updates all the references will point to the older `{{[Public IP Address in Decimal]}}` variable.

Reply:
1.1.0

Reply:
Revert this - the original was correct